tests/analysis/polarization/test_polarization.py

```diff
@@ -138,6 +138,61 @@ def test_polarization_with_differentially_polarized_markers():
     assert_frame_equal(scores, expected, check_exact=False, atol=1e-3)
 
 
+def test_polarization_with_min_marker_count():
+    # Set seed to get same graph every time
+    graph = create_randomly_connected_bipartite_graph(
+        n1=50, n2=100, p=0.1, random_seed=2
+    )
+
+    random.seed(1)
```

Comment:
I think it would be good to use a specific random number generator here instead of setting the global seed. See the numpy docs here: https://numpy.org/doc/stable/reference/random/index.html#quick-start

Reply:
Good suggestion!